suggested-content/word-2-vec-service.py

```diff
@@ -27,6 +31,14 @@ def documentCheckPhrases():
     response = checkPhrases(grams)
     return jsonify(results = response)
 
+@app.route('/doc', methods=['POST'])
+def similarDocs():
+    doc = request.json["doc"]   
```

Comment:
Yeah this is an endpoint to process to a POST request (from composer say) which contains a body of text and can be analysed against our documents in memory. The method for processing the CAPI file is called loadDocuments